skyfield/vectorlib.py

```diff
@@ -69,6 +69,9 @@ def at(self, t):
             raise ValueError('please provide the at() method with a Time'
                              ' instance as its argument, instead of the'
                              ' value {0!r}'.format(t))
+        if getattr(t.tt, 'shape', ()) and t.tt.shape == (0,):
+            raise ValueError('the time object provided to the at() method is '
```

Comment:
Could you outline why this should raise an error? Thanks! (I ask because there are lots of values that could wind up empty in Skyfield, but I'm not sure how a normal user would come up with an empty `Time` object to begin with, and if we add this check then we'll have to face the question of whether every Skyfield call everywhere needs a check for empty inputs, and that's a path I'm wary of because it's several dozen more lines of code for an error I'm not sure yet how real world users would run into.)

Reply:
I was trying to think of any problems empty objects could cause, and I guess I was being a little overzealous. It's not necessary, so if you have doubts leave it out.

Reply:
Understood! Let me know if you'd like to remove it when you update the pull request to change the repr's we discussed above, or if I should try pulling it out as I merge.